client/package.json

```diff
@@ -29,7 +29,7 @@
   },
   "scripts": {
     "start": "vite --port 3000",
-    "build": "tsc -b && vite build",
+    "build": "tsc -b && vite build --watch",
```

Comment:
--watch shoudn't be here. It messes up the CI pipeline and runs the github action for infinite duration